pyuvsim/utils.py

```diff
@@ -11,7 +11,7 @@
 try:
     import psutil
     HAVE_PSUTIL = True
-except ImportError:
+except ImportError:  # pragma: no cover
     HAVE_PSUTIL = False
```

Comment:
Oh, this whole block should be replaced with just `import psutil`. If it's a required dependency we don't need to do this dance.

Reply:
Oh yeah, good catch!

Reply:
It doesn't look like you made this change yet.

Reply:
huh. I swear I did. Maybe I didn't CTRL+S

Reply:
OK it's done now.